canonical-websites/www.ubuntu.com
# Use default vanilla pull quote pattern

## Done

Fix style of pull quotes on `/desktop/partners` to use default vanilla pattern

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/partners>
- See that pull quotes match the [Vanilla pattern](https://docs.vanillaframework.io/en/patterns/pull-quote)


## Issue / Card

Fixes #3705
